### DISCUSSION TITLE
Global Vim-like navigation option?

### DISCUSSION THREAD

**Discussion by TomJGooding** [Textualize/textual#2185] [Ideas]
Hopefully I'm not the only one, but I often find myself hitting the `hjkl` keys a few times in Textual before remembering that's not how the navigation works!

I'm honestly not sure this idea is feasible given the scope, but somehow having a 'global' option to use Vim-like bindings for the navigation would be a great feature!

**Comment by willmcgugan** [comment]
At some point we will have a config file that will allow you to override default bindings. So yes, you could set vim bindings on an app, or even for all Textual apps.

**Reply by TomJGooding** [reply]
Thanks Will - I should have known you'd already considered something like that!

(I really wasn't expecting a reply this evening... the work put into Textual is incredible but I hope you all do switch off and don't risk burnout!)
